src/logger/loggerApi.c

```diff
@@ -307,6 +307,13 @@ int api_sampleData(Serial *serial, const jsmntok_t *json){
 	return API_SUCCESS_NO_RETURN;
 }
 
+int api_heartBeat(Serial *serial, const jsmntok_t *json){
```

Comment:
this is following the style in the rest of the file; I'd prefer to sweep through and change everything at once in a separate style commit.